spikeinterface/core/binaryrecordingextractor.py

```diff
@@ -11,6 +11,37 @@
 
 
 class BinaryRecordingExtractor(BaseRecording):
+    """
+    RecordingExtractor for a binary format
+
+    Parameters
+    ----------
+    files_path: str or Path or list
+        Path to the binary file
+    sampling_frequency: float
+        The sampling frequncy
+    num_chan: int
+        Number of channels
+    dtype: str or dtype
+        The dtype of the binary file
+    time_axis: int
+        The axis of the time dimension (default 0: F order)
+    channel_ids: list (optional)
+        A list of channel ids
+    file_offset: int (optional)
+        Number of bytes in the file to offset by during memmap instantiation.
+    gain_to_uV: float or array-like (optional)
+        The gain to apply to the traces
+    offset_to_uV: float or array-like
+        The offset to apply to the traces
+    is_filtered: bool
+        If True, the recording is assumed to be filtered
```

Comment:
OK gain_to_uV/offset_to_uV but not OK for is_filtered.
It has to be ouside this I think

Reply:
Long signature are a pain

Reply:
I disagree. Since the binary file can be anything, it makes sense to have an easier way to set the is_filtered annotation